opentech/apply/funds/templates/funds/includes/delegated_form_base.html

```diff
@@ -20,6 +20,24 @@
         {% endif %}
     {% endfor %}
 
+    {% if report_frequency_form %}
```

Comment:
I didn't want to do this, adding a highly specific implementation into the base template will get messy long term and totally disassociates it from the only place it gets used.

Is there no way to move this out to somewhere near here: https://github.com/OpenTechFund/opentech.fund/pull/1662/files#diff-407ee86547708479420457906b7a732aR19

Reply:
Moved to the top of the modal which goes against the design, but don't think there's a neat way to achieve what the design asks for. Adding it all via JS seems overkill.

Reply:
Thanks Chris, i like that - lets review it on dev